wum-live-sync-automation/docker/Dockerfile

```diff
@@ -0,0 +1,174 @@
+# ------------------------------------------------------------------------
+#
+# Copyright 2016 WSO2, Inc. (http://wso2.com)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+# ------------------------------------------------------------------------
+
+FROM ubuntu:trusty
+
+RUN \ 
+apt-get -q update && apt-get install -y openssh-server software-properties-common git ant curl unzip xvfb dbus-x11 ttf-ancient-fonts \
+&& apt-get clean \
+&& rm -rf /var/lib/apt/lists/* \
+&& mkdir /var/run/sshd
+
+
+RUN \
+apt-get update \
+&& apt-get --yes --force-yes install firefox \
+&& apt-get --yes --force-yes install firefox=28.0+build2-0ubuntu2 \
+&& apt-mark hold firefox \
+&& apt-get clean \
+&& rm -rf /var/lib/apt/lists/*
+
+RUN \
+mkdir -p /build/software/maven \
+&& wget -P /build/software/maven https://archive.apache.org/dist/maven/maven-3/3.0.5/binaries/apache-maven-3.0.5-bin.tar.gz \
+&& tar -xvzf /build/software/maven/apache-maven-3.0.5-bin.tar.gz --directory /build/software/maven \
+&& rm /build/software/maven/apache-maven-3.0.5-bin.tar.gz
```

Comment:
I believe we probably do not need older maven versions for WUM supported products.